projects/pt1/python/torch_mlir_e2e_test/test_suite/matmul.py

```diff
@@ -289,3 +289,94 @@ def forward(self, x, y):
 def AtenMmQuint8_basic(module, tu: TestUtils):
     module.forward(tu.randint(3, 4, low=-128, high=127).to(torch.int8),
                    tu.randint(4, 3, low=-128, high=127).to(torch.int8))
+    
+# ==============================================================================
+
+class AtenLinalgCrossInt(torch.nn.Module):
```

Comment:
Does the lowering work with dynamic dimensions? If so, can you add at least one test case with dynamic dims?